frontend/src/components/pages/flatPage/useFlatPage.tsx

```diff
@@ -14,13 +17,38 @@ export const useFlatPage = (flatPageUrl: string | undefined) => {
 
   const commonDictionaries = useQueryCommonDictionaries(language);
 
-  const { data, refetch, isLoading, error } = useQuery<FlatPageDetails, Error>(
+  const {
+    data: flatPage,
+    refetch,
+    isLoading,
+    error,
+  } = useQuery<FlatPageDetails, Error>(
     ['flatPageDetails', id, language],
     () => getFlatPageDetails(id, language, commonDictionaries),
     {
       enabled: isUrlString(flatPageUrl) && commonDictionaries !== undefined,
       staleTime: ONE_DAY,
     },
   );
-  return { id, flatPage: data, refetch, isLoading, error, path };
+  const suggestions = getSuggestionsFromContent(flatPage?.content ?? '');
+
+  const activitySuggestionIds = suggestions.flatMap(suggestion =>
+    'ids' in suggestion ? suggestion.ids : [suggestion.type],
```

Comment:
```suggestion
    suggestion.ids ?? [suggestion.type],
```

Reply:
My code is a bit tricky because I have a little problem with the `suggestion` type, which thinks `ids` doesn't exist, but it does. :eyes:

Reply:
Thx for the suggestion :pray: